integration/integration.go

```diff
@@ -234,6 +241,14 @@ func (i *Integration) Logger() log.Logger {
 	return i.logger
 }
 
+// CreateUniqueID will generate an md5 string from integration arguments
+// to unique identify the integration instance.
+func (i *Integration) CreateUniqueID() string {
+	h := md5.New()
```

Comment:
not big deal, as integrations won't have enough instances to collide, but in case they did this may be a good option https://github.com/google/uuid